### PR TITLE
linux: fix appimage

### PR DESCRIPTION
**Pull request by daiyam**
This PR is replacing the old `pkg2appimage` binary with its latest version.
It's also generating the `zsync` file so that the appimage becomes updateable.

It's fixing the issue #330.

### CODE REVIEW

File: create_appimage.sh [hunk 1]
```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
+
+cd ..
+
 if [[ "$VSCODE_ARCH" == "x64" ]]; then
   # install a dep needed for this process
   sudo apt-get install desktop-file-utils
 
-  cd ..
-  export DOCKER_BUILD=1
```
Comment (stripedpajamas) [inline create_appimage.sh:7]: Does the latest version of the appimage utility auto detect it's running in a docker container now? This was required before.

Comment (daiyam) [reply create_appimage.sh:7]: It seems so with [`grep docker /proc/1/cgroup >/dev/null && export DOCKER_BUILD=1 || true`](https://github.com/AppImage/pkg2appimage/blob/23e510b394fc28d24386e2eba076132a646f456c/functions.sh#L33).

But I'm having an issue the imagemagick's `convert` which are failing the packaging. And the error `Cannot mount AppImage, please check your FUSE setup.` is weird...

**Comment by stripedpajamas** [comment]
See https://github.com/VSCodium/vscodium/issues/532 for what the issue was last time

**Comment by daiyam** [comment]
OK, an AppImage is launched to make the AppImage. So I understand the error and know what to test. Thx

**Comment by daiyam** [comment]
The issue was that `convert` is also provided by `pkg2appimage` but as an AppImage which fail when called in Docker...
So I've replaced the path to use the system's `convert` which is a regular binary file.

**Review by stripedpajamas — APPROVED** [review]

**Pull request merged**